features/weak-references.yml

```diff
@@ -0,0 +1,14 @@
+name: Weak references
+description: The `WeakRef` and `FinalizationRegistry` objects manage weakly held objects and define cleanup operations for garbage-collected objects.
+spec:
+  - https://tc39.es/ecma262/multipage/managing-memory.html#sec-managing-memory
+snapshot: ecmascript-2021
+group: javascript
+compat_features:
+  - javascript.builtins.FinalizationRegistry
+  - javascript.builtins.FinalizationRegistry.FinalizationRegistry
+  - javascript.builtins.FinalizationRegistry.register
+  - javascript.builtins.FinalizationRegistry.unregister
+  - javascript.builtins.WeakRef
+  - javascript.builtins.WeakRef.WeakRef
+  - javascript.builtins.WeakRef.deref
```

Comment:
For the record, I wasn't 100% sure these needed to be together at first, but the original proposal illustrates their combined application, which convinced me: https://github.com/tc39/proposal-weakrefs?tab=readme-ov-file#using-weakref-objects-and-finalizationregistry-objects-together.